### DISCUSSION TITLE
Sub Button Styling - Apply current light color

### DISCUSSION THREAD

**Discussion by a-bianucci** [Clooos/Bubble-Card#814] [Questions about config, custom styles and templates]
Hello,

I'm familiar with how to style sub buttons, including depending on the state.  Is there a var that can be used to ensure that the sub button background-color matches the current light color.   The typical use of other var's that I leverage with other cards do not seem to apply here.

Essentially if you think of a light being a slider card, it shows the current color of the light as the background.  I'm trying to leverage whatever sets that as the background color (or icon color) on a sub button.

**Comment by MrBearPresident** [comment]
Doesn't the sub-button does this already?

---

<sub>If this solves your issue, please put this on solved.</sub>
